improve user interaction, re-factor current methods

NimSum opened the pull request:
Re-factored some of the methods within game class, re-factored some of the domUpdate methods, improved on the button functionality, and animated prize and portal on click

NimSum committed:
improve user interaction, re-factor current methods

raechelo committed:
Start of methods for scoring players, wheel now instantiates upon each spin. Animations corrected and inputs now disappear and reappear with user navigation

raechelo commented on src/sass/_wheel.scss:
```diff
@@ -22,22 +23,57 @@ main aside {
+  animation-duration: 3s;
+}
+
+@keyframes spin-prize {
```
i think this is an awesome animation but i think we should define it in our variables sass file with the other animation!

NimSum replied:
Fixed! moved keyframs into sass vars file

NimSum committed:
Merge pull request #28 from NimSum/score-test-branch

Start of methods for scoring players, wheel now instantiates upon…

NimSum committed:
Fix merge conflicts

NimSum committed:
Move animation into var sass, animation adjustment

NimSum commented:
Changes with the previously merged branch:
- Merged domUpdate methods that were in conflict, including duplicate event listeners
- Moved wheel spin inside of game(generatePrize) and added a new prob called current prize
- Moved answer validation invocation within game class(so we can alter the player score and move rounds within the game class) => if you have a diff approach in mind lmk!
 Theres prob more i forgot to include on this list, ask me if anything seems off!
